cime_config/config_compsets.xml

```diff
@@ -53,31 +58,13 @@
     <lname>2000_CAM%DEV%LT%NORESM%TROPMAM4_CLM50%SP_CICE%PRES_DOCN%DOM_MOSART_SGLC_SWAV</lname>
   </compset>
 
-  <compset>
-    <alias>NF2000devL32</alias>
-    <lname>2000_CAM%DEV%NORESM_CLM50%SP_CICE%PRES_DOCN%DOM_MOSART_SGLC_SWAV</lname>
-  </compset>
-  <compset>
-    <alias>NF2000osloL32</alias>
-    <lname>2000_CAM%DEV%NORESM%CAMoslo_CLM50%SP_CICE%PRES_DOCN%DOM_MOSART_SGLC_SWAV</lname>
-  </compset>
-  <compset>
-    <alias>NF2000tropmam4l32</alias>
-    <lname>2000_CAM%DEV%NORESM%TROPMAM4_CLM50%SP_CICE%PRES_DOCN%DOM_MOSART_SGLC_SWAV</lname>
-  </compset>
-
-  <compset>
-    <alias>NF2000climo</alias>
-    <lname>2000_CAM60%NORESM_CLM50%SP_CICE%PRES_DOCN%DOM_MOSART_SGLC_SWAV</lname>
-  </compset>
-
   <compset>
     <alias>NF1850</alias>
-    <lname>1850_CAM60%NORESM_CLM51%FATES-SP_CICE%PRES_DOCN%DOM_MOSART_SGLC_SWAV</lname>
+    <lname>1850_CAM%DEV%LT%NORESM_CLM51%SP_CICE%PRES_DOCN%DOM_MOSART_SGLC_SWAV</lname>
```

Comment:
For clarity, should we be explicit about the chemistry / aerosol suite for all of these compsets?

Reply:
Good point.

Reply:
I don't understand what you are looking at. There is no FATES in config_compsets.xml.

Reply:
The other compsets explicitly mention `TROPMAM4` or `CAMoslo`. Rather than rely on defaults, what chemistry is intended for this compset? I'm looking at CAM, not CLM.